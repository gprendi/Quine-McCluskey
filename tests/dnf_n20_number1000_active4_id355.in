d 20
00000000000000000000
00100000000000000000
00100010000001000000
00100011000001000000
00000010000000000000
00100011000000000000
00000001000001000000
00000000000001000000
00100000000001000000
00100001000000000000
00000010000001000000
00100010000000000000
00100001000001000000
00000001000000000000
00000011000001000000
00000011000000000000
