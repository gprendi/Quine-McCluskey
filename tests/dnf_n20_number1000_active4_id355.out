p dnf 20 0
-1 -2 -4 -5 -6 -9 -10 -11 -12 -13 -15 -16 -17 -18 -19 -20 0
